src/Components/Boolist/NewBookForm.js

```diff
@@ -25,15 +26,31 @@ const NewBookForm = () => {
     setInputs(initialInputs);
   };
   return (
-    <form onSubmit={handleClick}>
+    <form onSubmit={handleClick} className="form">
       <hr />
       <h3>Add New Book</h3>
       <div className="formInput">
         <input type="text" id="title" onChange={handleChange} name="title" value={input.title} placeholder="Book Title" />
         <br />
         <input type="text" id="name" onChange={handleChange} name="author" value={input.author} placeholder="Author" />
         <br />
-        <button type="submit" onClick={handleClick}>ADD BOOK</button>
+        <select
+          id="category"
+          className="selectOption"
+          name="category"
+          value={input.category}
+          onChange={handleChange}
+          style={{ paddingRight: '5px' }}
+        >
+          <option value="" disabled selected hidden className="selectCat">--Category--</option>
+          <option value="fiction">Fiction</option>
+          <option value="non-fiction">Non-Fiction</option>
+          <option value="mystery">Mystery</option>
+          <option value="sci-fi">Romance</option>
+          <option value="sci-fi">Fantasy</option>
+        </select>
+        <br />
+        <button type="submit" onClick={handleClick} className="btn">ADD BOOK</button>
       </div>
     </form>
```

Comment:
- [ ] @Luchy-B great job on your design however  the form occupies larger width than in the design also the button is small kindly increase its size and ensure it's color matches the design given,


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/232139221-ef93c3f4-e2e2-48a0-ad15-9e1b9fb1f1bd.png) |  ![image](https://user-images.githubusercontent.com/61408860/232139283-d91e9c17-db53-4577-ac39-702b195a6fc8.png)